app/src/test/java/org/simple/clinic/contactpatient/ContactPatientInitTest.kt

```diff
@@ -43,9 +44,19 @@ class ContactPatientInitTest {
 
   @Test
   fun `when the screen is restored, do not load the patient profile, latest appointment and current facility if they are already loaded`() {
+    val facility = TestData.facility(
+        uuid = UUID.fromString("251deca2-d219-4863-80fc-e7d48cb22b1b"),
+        name = "PHC Obvious",
+        facilityConfig = FacilityConfig(
+            diabetesManagementEnabled = true,
+            teleconsultationEnabled = false
+        )
+    )
+
     val model = defaultModel()
         .patientProfileLoaded(TestData.patientProfile(patientUuid = patientUuid))
         .overdueAppointmentLoaded(Optional.empty())
+        .currentFacilityLoaded(currentFacility = facility)
```

Comment:
Ideally this change should be done in the commit where we are changing Init block

Reply:
Just an FYI, we haven't added this method yet before this commit. So this commit alone wouldn't compile. But since we are squashing commits, it's fine now. But something to be careful about.